ratslam_view_template:
  ros__parameters:
    image_crop_y_max: 150
    template_x_size: 60
    template_y_size: 20
    vt_shift_match: 4
    vt_step_match: 1
    vt_match_threshold: 0.03
    # draw
    enable: true
    vt_window_width: 416
    vt_window_height: 480

ratslam_pose_cells:
  ros__parameters:
    topic_root: "irat_red"
    pc_dim_xy: 11
    pc_vt_inject_energy: 0.1
    pc_cell_x_size: 0.015
    # draw
    enable: true

ratslam_experience_map:
  ros__parameters:
    topic_root: "irat_red"
    exp_loops: 20
    exp_initial_em_deg: 140
    # draw
    enable: true
    vt_window_width: 416
    vt_window_height: 480
    exp_map_size: 500
    posecells_size: 250